### PR TITLE
added pr labeler

### PR DESCRIPTION
**Pull request by hlts2**
### Description:
I added out labeling into pr.
Because the area label is given by labeler, so i give the label in consideration of type.

### Environment:

- Golang Version: 1.13.5
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.8.4

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/pr-auto-labeler.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            "type/ci"
+            "type/dependency"
+        )
+
+        echo $branch_name
+        for label in ${type_labels[@]}
+        do
+          if [ "`echo ${branch_name} | grep ${label##*/}`" ] ; then
+            echo $label
+            add_label ${label}
```
Comment (hlts2) [inline .github/workflows/pr-auto-labeler.yml]: :moemo:
It seems that the performance is better if you send it in bulk

File: .github/workflows/pr-auto-labeler.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            "type/security"
+            "type/test"
+            "type/ci"
+            "type/dependency"
+        )
+
+        echo $branch_name
+        for label in ${type_labels[@]}
+        do
+          if [ "`echo ${branch_name} | grep ${label##*/}`" ] ; then
```
Comment (rinx) [inline .github/workflows/pr-auto-labeler.yml]: It's same as `[ "${branch_name%%/*}" = "${label#type/}" ]`

File: .github/workflows/pr-auto-labeler.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+name: Run PR Auto Labeler
```
Comment (rinx) [inline .github/workflows/pr-auto-labeler.yml]: You can merge it to labeler.yml.

Comment (hlts2) [reply .github/workflows/pr-auto-labeler.yml]: oh, thank you!

File: .github/workflows/labeler.yml [hunk 4]
```diff
@@ -9,3 +9,62 @@ jobs:
...
+        echo "branch name: " $branch_name
+        for label in ${type_labels[@]}
+        do
+          if [ "${branch_name%%/*}" = "${label#type/}" ] ; then
+            send_labels+=(\"$label\")
+          fi
+        done
+
+        size=0
+        diff=`git diff --shortstat origin/master...${branch_name} ':!./*\.yaml' ':!./*\.yml' ':!./*\.svg' ':!./apis/*'`
```
Comment (rinx) [inline .github/workflows/labeler.yml]: Please exclude `/design` directory.

Comment (rinx) [reply .github/workflows/labeler.yml]: It contains `.drawio` file.

Comment (hlts2) [reply .github/workflows/labeler.yml]: oh..:bow: thank you!!
I wil fix it.

Comment (hlts2) [reply .github/workflows/labeler.yml]: fixed it!

**Comment by rinx** [comment]
@hlts2 Thanks! Great job!

**Pull request merged**